conf/evolutions/default/1.sql

```diff
@@ -3,14 +3,17 @@
 
 # --- !Ups
 
-create table users (
+create table abh_user (
   email                     varchar(255) not null,
-  surname                   varchar(255),
-  name                      varchar(255),
+  last_name                 varchar(255),
+  first_name                varchar(255),
   password                  varchar(255),
-  token                     varchar(255),
+  auth_token                varchar(255),
```

Comment:
Do we really need varchar of size 255 for all these columns? Think of all unused, but reserved memory space.